csrankings-s.csv

```diff
@@ -1982,6 +1982,7 @@ Syed M. S. Islam,University of Western Australia,http://www.ecm.uwa.edu.au/conta
 Syed Mohammed Shamsul Islam,University of Western Australia,http://www.ecm.uwa.edu.au/contact/profiles?type=profile&dn=cn%3DSyed%20Islam%2Cou%3DSchool%20of%20Computer%20Science%20and%20Software%20Engineering%2Cou%3DFaculty%20of%20Engineering%5C2C%20Computing%20and%20Mathematics%2Cou%3DFaculties%2Co%3DThe%20University%20of%20Western%20Australia,2oSH1wgAAAAJ
 Syed Rafiul Hussain,Pennsylvania State University,https://syed-rafiul-hussain.github.io,mgjniiwAAAAJ
 Syed Zulqarnain Gilani,University of Western Australia,http://www.ecm.uwa.edu.au/research/postgraduate_profiles?syed.zulqarnain.gilani,0uXgj0oAAAAJ
+Syed Furqan Qadri,Shenzhen University,https://www.researchgate.net/profile/Syed-Furqan-Qadri,en&authuser=1

```

Comment:
Malformed Google Scholar ID. Also, web page says he is a postdoc, and therefore not eligible for inclusion.